sheeprl/utils/utils.py

```diff
@@ -43,6 +43,14 @@ def __getstate__(self):
     def __setstate__(self, state):
         self.update(state)
 
+    @property
```

Comment:
It would be better to have a method like `as_dict` to get a plain dictionary out of a `dotdict`